ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixParticipantTest.java

```diff
@@ -63,19 +65,97 @@
 public class HelixParticipantTest {
   private final MockHelixManagerFactory helixManagerFactory;
   private final Properties props;
+  private final String clusterName = "HelixParticipantTestCluster";
 
   public HelixParticipantTest() throws Exception {
     List<com.github.ambry.utils.TestUtils.ZkInfo> zkInfoList = new ArrayList<>();
     zkInfoList.add(new com.github.ambry.utils.TestUtils.ZkInfo(null, "DC0", 2199, false));
     JSONObject zkJson = constructZkLayoutJSON(zkInfoList);
     props = new Properties();
     props.setProperty("clustermap.host.name", "localhost");
-    props.setProperty("clustermap.cluster.name", "HelixParticipantTestCluster");
+    props.setProperty("clustermap.cluster.name", clusterName);
     props.setProperty("clustermap.datacenter.name", "DC0");
     props.setProperty("clustermap.dcs.zk.connect.strings", zkJson.toString(2));
     helixManagerFactory = new MockHelixManagerFactory();
   }
 
+  private ReplicaId createMockReplicaId(String partitionIdString) {
+    ReplicaId replicaId = Mockito.mock(ReplicaId.class);
+    PartitionId partitionId = Mockito.mock(PartitionId.class);
+    Mockito.when(partitionId.toPathString()).thenReturn(partitionIdString);
+    Mockito.when(replicaId.getPartitionId()).thenReturn(partitionId);
+    return replicaId;
+  }
+
+  private void listIsExpectedSize(List list, int expectedSize, String name) {
+    assertNotNull(name + " is null", list);
+    assertEquals(name + " is not size " + expectedSize, expectedSize, list.size());
+  }
+
+  /**
+   * Tests setReplicaSealedState method for {@link HelixParticipant}
+   * @throws IOException
+   */
+  @Test
+  public void testSetReplicaSealedState() throws IOException {
+    //setup HelixParticipant and dependencies
+    String partitionId = "somePartitionId";
```

Comment:
Since `PartitionId` is a class, would prefer if this variable is named `partitionIdStr`. Makes it easy to follow code.